### ISSUE TITLE
About Initial release of annotations.

### ISSUE THREAD

**Issue by bob35buaa** [ZhengdiYu/SignAvatars#2]
Excellent work! I noticed that the provided annotations include only a portion of the SMPL-X annotations (specifically, only have the training set for the How2Sign dataset and the test set for the PHOENIX-2014T dataset). Could you share the full annotations? Thank you!

**Comment by ZhengdiYu** [comment]
Sorry for the late response. Our colleagues have been working on fixing the missing link for the data and we've been pushing forward the complete version of the dataset, which will be available soon. Thank you so much for your patience! : )
